### PR TITLE
update docs to match new cli updates

### PR DESCRIPTION
**Pull request by davemooreuws**
doc updates for nitric new and stack new commands

### CODE REVIEW

**Review by tjholm — DISMISSED** [review]

File: src/pages/reference/cli/stacks.mdx [hunk 1]
```diff
@@ -72,7 +147,7 @@ config:
     memory: 4096
 ```
 
-The provider has been updated from `aws`, `gcp`, and `azure` to `nitric/aws@0.24.0`, `nitric/gcp@0.24.0`, and `nitric/azure@0.24.0` respectively. This change enables locking a version for your deployments, as it was previously tied to the CLI version. This change will also enable creation of your own [custom deployment providers](/reference/providers/custom/building-custom-provider).
+The provider has been updated from `aws`, `gcp`, and `azure` to `nitric/aws@x.x.x`, `nitric/gcp@x.x.x`, and `nitric/azure@x.x.x` respectively. This change enables locking a version for your deployments, as it was previously tied to the CLI version. This change will also enable creation of your own [custom deployment providers](/reference/providers/custom/building-custom-provider).
```
Comment (tjholm) [inline src/pages/reference/cli/stacks.mdx:150]: This syntax might be a better way to support versioning for `@latest` going forward.

e.g. 0.35.x to get the latest patch revision of version 0.35.

File: src/pages/guides/getting-started/nodejs/expressjs.mdx [hunk 2]
```diff
@@ -22,12 +22,13 @@ To complete this guide you'll need the following:
...
-? Choose a template: official/JavaScript - Starter
+nitric new express-example "official/JavaScript - Starter"
+```
+
+Then install dependencies and add express:
 
-# install depedencies and add express
-cd express-example
+```
+cd fastify-example
```
Comment (HomelessDinosaur) [inline src/pages/guides/getting-started/nodejs/expressjs.mdx:31]: Shouldn't this be express?
```suggestion
cd express-example
```

File: src/pages/reference/cli/stacks.mdx [hunk 3]
```diff
@@ -18,25 +18,69 @@ nitric stack new
...
 The stack definition will be created in the root of your project as a YAML file prefixed with `nitric-` e.g. `nitric-aws-stack.yaml`
 
-```yaml
+<CodeGroup>
+
+```yaml{{ title: "AWS" }}
 name: my-aws-stack
-provider: nitric/aws@0.24.0
+# Using @latest in production can lead to breaking changes. Pin to a specific version for stability.
+provider: nitric/aws@latest
```
Comment (HomelessDinosaur) [inline src/pages/reference/cli/stacks.mdx:53]: Are we keeping this as latest or doing `x.x.x` referring to Tim's comment earlier.

**Review by HomelessDinosaur — CHANGES_REQUESTED** [review]

File: src/pages/reference/cli/stacks.mdx (hunk 3 above)
Comment (davemooreuws) [reply src/pages/reference/cli/stacks.mdx:53]: keeping as latest for now

File: src/pages/guides/getting-started/nodejs/stripe.mdx [hunk 4]
```diff
@@ -32,7 +32,7 @@ Create a project name, select the TypeScript template and choose the default glo
 
 ```bash
 ? What is the name of the stack? nitric-stripe
```
[the remark this answers is not in the document]
Comment (davemooreuws) [reply src/pages/guides/getting-started/nodejs/stripe.mdx]: This is removed anyway. Resolved

**Review by HomelessDinosaur — APPROVED** [review]

**Pull request merged**